### DISCUSSION TITLE
leptos_axum 0.0.4 doesn't seem to be updated on crates.io/docs.rs

### DISCUSSION THREAD

**Discussion by foodornt** [leptos-rs/leptos#183] [General]
On [docs.rs](https://docs.rs/leptos_axum/latest/src/leptos_axum/lib.rs.html#1-286): only 286 lines
On [github](https://github.com/gbj/leptos/blob/main/integrations/axum/src/lib.rs): already 422 lines

Some changes need to be pushed to crates.io I guess? I'm not sure how their update mechanism works

**Comment by gbj** [comment]
Thanks, I've updated it.
